.travis.yml

```diff
@@ -56,7 +56,7 @@ script:
 - export TAICHI_REPO_DIR=$TRAVIS_BUILD_DIR
 - export PYTHONPATH=$TAICHI_REPO_DIR/python
 - export PATH=$TAICHI_REPO_DIR/bin:$PATH
-- ti test_verbose && cd python && $PYTHON build.py try_upload
+- ti test -v && ti test_cpp -v && cd python && $PYTHON build.py try_upload
```

Comment:
I suggest that we merge `test_cpp` into test: https://github.com/taichi-dev/taichi/pull/601#issuecomment-599212596